packages/terra-clinical-error-view/README.md

```diff
@@ -4,7 +4,8 @@
 [![NPM version](http://img.shields.io/npm/v/terra-clinical-error-view.svg)](https://www.npmjs.org/package/terra-clinical-error-view)
 [![Build Status](https://travis-ci.org/cerner/terra-clinical.svg?branch=master)](https://travis-ci.org/cerner/terra-clinical)
 
-The Terra ErrorView is a display presented with an error message and location for a button.
+The Terra ErrorView is a display presented with an error message and location for a button.  This component has been deprecated in
```

Comment:
Could we bold the 'deprecated' sentence for all these readmes?

Reply:
deprecated is now in bold